modules/exploits/linux/http/mvpower_dvr_shell_exec.rb

```diff
@@ -60,6 +60,7 @@ def check
     begin
       fingerprint = Rex::Text::rand_text_alpha(rand(10) + 6)
       res = send_request_cgi(
+        'method' => "GET",
         'uri' => "/shell?echo+#{fingerprint}",
```

Comment:
I'm sorry for that and thanks a lot for merging my PR. I will continue reading about how everything works and won't issue any PR without proper knowledge. @wvu-r7